scripts/host/php-storm-configs/<host_name>.iml

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure why this file was removed, but that's ok, I will restore it after accepting PR.